src/os/bluestore/BlueStore.h

```diff
@@ -3353,6 +3353,8 @@ class BlueStore : public ObjectStore,
   }
 
   /// methods to inject various errors fsck can repair
+  int get_shared_blob(const std::string& key,
+		       ceph::buffer::list& bl);
```

Comment:
Comment might need to be moved below `get_shared_blob` as it doesn't reflec `get_shared_blob`

Reply:
Fixed